### PR TITLE
docs: Remove extra exports

### PR DESCRIPTION
**Pull request by ling1726**
Doc examples should only have one export.

### CODE REVIEW

File: packages/react-components/react-tabs/stories/Tabs/TabListDefault.stories.tsx [hunk 1]
```diff
@@ -1,29 +1,263 @@
...
+  Button,
+  Menu,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
+  Tab,
+  TabList,
+} from '@fluentui/react-components';
+import { useIsOverflowItemVisible, useOverflowMenu, Overflow, OverflowItem } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-tabs/stories/Tabs/TabListDefault.stories.tsx:15]: You can put both of these under one import statement

```suggestion
import {
  makeStyles,
  mergeClasses,
  shorthands,
  tokens,
  useIsOverflowItemVisible,
  useOverflowMenu,
  Button,
  Menu,
  MenuItem,
  MenuList,
  MenuPopover,
  MenuTrigger,
  Overflow,
  OverflowItem,
  Tab,
  TabList,
} from '@fluentui/react-components';
```

**Review by khmakoto — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `60.588 kB`<br />`16.969 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `188.355 kB`<br />`52.98 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `34.751 kB`<br />`11.453 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `118.221 kB`<br />`36.568 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             |  `121.29 kB`<br />`37.071 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.069 kB`<br />`2.053 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/52fad61a1e6c29e8e815fd74bc66f06a100710e3'>52fad61a1e6c29e8e815fd74bc66f06a100710e3</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/Avatar-mount_base.html">1292</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/Avatar-mount.html">1288</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/Button-mount_base.html">938</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/Button-mount.html">925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/FluentProvider-mount_base.html">1531</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/FluentProvider-mount.html">1535</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/FluentProviderWithTheme-mount_base.html">573</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/FluentProviderWithTheme-mount.html">574</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">543</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">546</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">576</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">589</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/InfoButton-mount_base.html">517</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/InfoButton-mount.html">519</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/MakeStyles-mount_base.html">1951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/MakeStyles-mount.html">1928</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/Persona-mount_base.html">2828</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/Persona-mount.html">2789</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/SpinButton-mount_base.html">2396</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26567/perf-test-react-components/SpinButton-mount.html">2392</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-menu/stories/MenuList/MenuListCheckboxItems.stories.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const CutIcon = bundleIcon(CutFilled, CutRegular);
 const PasteIcon = bundleIcon(ClipboardPasteFilled, ClipboardPasteRegular);
```
Comment (fabricteam) [inline packages/react-components/react-menu/stories/MenuList/MenuListCheckboxItems.stories.tsx:15]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26567&commitId=0f8df00a87e65e9c8c72dc008178e7db18e90199&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1831 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|1| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

**Review by layershifter — APPROVED** [review]

**Pull request merged**